toolchains/FreeBSD-x64.cmake

```diff
@@ -0,0 +1,18 @@
+if(NOT _VCPKG_FREEBSD_TOOLCHAIN)
+    set(_VCPKG_FREEBSD_TOOLCHAIN 1)
+endif()
+
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "FreeBSD")
+    set(CMAKE_CROSSCOMPILING OFF CACHE BOOL "")
+endif()
+
+set(CMAKE_SYSTEM_NAME FreeBSD CACHE STRING "")
+
+# set(THREADS_PREFER_PTHREAD_FLAG ON)
```

Comment:
can this be cleaned up?

Reply:
Prefer no file (since it's unnecessary), commented detritus for future reference, or an empty file?

Reply:
This file was removed.